### PR TITLE
fix header, add about and contact

### PR DESCRIPTION
**Pull request by Zakintaliban**
Adding a link to the logo, removing home and the link itself, in header.js.
Adding a contact in header.js.
Adding social media in contact.js
Adding a few paragraphs in about.js.

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -7,11 +7,13 @@ import "./header.css"
 const Header = ({ siteTitle }) => (
   <header className="header">
     <div className="header__container">
-      <h1 className="header__title">High Skill Masters</h1>
+      <Link to="/">
+        <a className="header__title">High Skill Masters</a>
```
Comment (rahmanfadhil) [inline src/components/header.js]: @Zakintaliban if you use Link component from Gatsby, you don't need to nest it inside a link.

File: src/pages/contact.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from "react"
+
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+const Contact = () => (
```
Comment (rahmanfadhil) [inline src/pages/contact.js:7]: @Zakintaliban  Add image to this page

**Review by rahmanfadhil — CHANGES_REQUESTED** [review]

File: src/components/header.js (hunk 1 above)
Comment (Zakintaliban) [reply src/components/header.js]: k

File: src/pages/contact.js (hunk 2 above)
Comment (Zakintaliban) [reply src/pages/contact.js:7]: k

**Review by Zakintaliban — COMMENTED** [review]
I already fix the image and a.

**Review by rahmanfadhil — APPROVED** [review]
LGTM!

**Pull request merged**